### PR TITLE
OC-10836

### PR DESCRIPTION
**Pull request by gushil**
## Checklist

1. [ ] If you've added code that should be tested, add tests
2. [ ] If you've changed APIs, update (or create!) the documentation
3. [ ] Ensure the tests pass
4. [ ] Make sure your code lints and you followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [ ] If this is a big feature, make sure to prefix the title with `Feature:` and add a thorough description for non-dev folk

### CODE REVIEW

File: jsapp/xlform/src/view.rowDetail.coffee [hunk 1]
```diff
@@ -123,6 +121,14 @@ module.exports = do ->
       input_label = input_label
       @field """<input type="checkbox" name="#{key}" id="#{cid}"/> <label for="#{cid}">#{input_label}</label>""", cid, key_label
 
+    radiobutton: (cid, key, options, key_label = key, default_value = '') ->
```
Comment (yogi-oc) [inline jsapp/xlform/src/view.rowDetail.coffee]: Can you please make this camelcase? like radioButton?

File: jsapp/xlform/src/view.rowDetail.coffee [hunk 2]
```diff
@@ -123,6 +121,14 @@ module.exports = do ->
...
+      buttons = ""
+      for option in options
+        buttons += """<input type="radio" name="#{key}" id="option_#{option.label}" value="#{option.value}">"""
+        buttons += """<label id="label_#{option.label}" for="#{option.label}">#{option.label}</label>"""
+
+      @field buttons, cid, key_label
+
     dropdown: (cid, key, values, key_label = key) ->
```
Comment (yogi-oc) [inline jsapp/xlform/src/view.rowDetail.coffee]: same if you can make it dropDown that would be great

File: jsapp/xlform/src/view.rowDetail.coffee (hunk 1 above)
Comment (gushil) [reply jsapp/xlform/src/view.rowDetail.coffee]: I already follow the current code convention. Should I continue changing into camelcase?

File: jsapp/xlform/src/view.rowDetail.coffee (hunk 2 above)
Comment (gushil) [reply jsapp/xlform/src/view.rowDetail.coffee]: dropdown is upstream code. Should I change that too?

**Comment by yogi-oc** [comment]
:+1:

**Pull request merged**